css/package.css

```diff
@@ -359,3 +359,6 @@ ul.tab-container {
     border-radius: 4px;
     width: 20%;
 }
+a {
+    color: #006ee5 !important;
```

Comment:
I believe the accessibility issue was that the color of version>= was too similar to the dependency name. If so why this change?

Reply:
Hi @JavierMatosD ,

This is the accessibility fix for home page URL highlighted in the picture.
![image](https://github.com/vcpkg/vcpkg.github.io/assets/69580789/b1edba2a-0e28-4cfc-ad2a-c02cfbdea0eb)

Reply:
Okk